### PR TITLE
ZER 127

### PR DESCRIPTION
**Pull request by BeforeBlackout**
Created new project on Google Cloud Platform
Created Oauth 2.0 Cliend ID (TXT file attached below)
Added to config omniauth configuration
Set up omniauth callbacks and method to access Google

## JIRA
* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=3177&projectKey=ZER&view=detail&selectedIssue=ZER-127)

## ToDo
Fix login with Google

## Testing section
-Oauth 2.0 Cliend ID-
[Cliend ID for Web Application .txt](https://github.com/ita-social-projects/ZeroWaste/files/8547539/Cliend.ID.for.Web.Application.txt)
Client_ID

-Demonstration-

logged_in

### CODE REVIEW

**Review by dkh — COMMENTED** [review]
Is it necessary to confirm the account?

**Comment by BeforeBlackout** [comment]
> Is it necessary to confirm the account?

Yeah, you`re right. I`ve fixed it

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -272,11 +272,13 @@
...
-  google_client_id = Rails.application.credentials.google_oauth2[:client_id]
-  google_client_secret = Rails.application.credentials.google_oauth2[:client_secret]
-  config.omniauth :google_oauth2, google_client_id, google_client_secret,
-                  scope: 'userinfo.email,userinfo.profile', skip_jwt: true
-
+  # google_client_id = Rails.application.credentials.google_oauth2[:client_id]
+  # google_client_secret = Rails.application.credentials.google_oauth2[:client_secret]
+  # config.omniauth :google_oauth2, google_client_id, google_client_secret,
+  #                 scope: 'userinfo.email,userinfo.profile', skip_jwt: true
+  GOOGLE_CLIENT_ID = '936253024035-pmkppsgge4psmcpserbdk8e05o7pd3ub.apps.googleusercontent.com'
```
Comment (dkh) [inline config/initializers/devise.rb:279]: One more thing, it's a big security issue to put store credentials in the public files

Comment (mozulevskyi) [reply config/initializers/devise.rb:279]: ＋

Comment (BeforeBlackout) [reply config/initializers/devise.rb]: Fixed

**Review by mozulevskyi — APPROVED** [review]

**Pull request merged**